h/traversal/roots.py

```diff
@@ -146,7 +146,11 @@ class BulkAPIRoot(RootFactory):
     """Root factory for the Bulk API."""
 
     # Currently only LMS uses this end-point
-    __acl__ = [(Allow, "client_authority:lms.hypothes.is", "bulk_action")]
+    __acl__ = [
+        (Allow, "client_authority:lms.hypothes.is", "bulk_action"),
+        # A cool and scalable solution...
+        (Allow, "client_authority:lms.ca.hypothes.is", "bulk_action"),
```

Comment:
So far I'm putting a pin in this one while I work out the rest. The variable client authority principals mapping to permissions doesn't really work great here.

I think we want an active Policy here (ala https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/security.html#implementing-acl-authorization), but I don't know exactly how that's actually done.